exchange/auction.go

```diff
@@ -60,11 +60,11 @@ func (a *auction) setRoundedPrices(priceGranularity openrtb_ext.PriceGranularity
 	a.roundedPrices = roundedPrices
 }
 
-func (a *auction) doCache(ctx context.Context, cache prebid_cache_client.Client, bids bool, vast bool, bidRequest *openrtb.BidRequest, ttlBuffer int64, defaultTTLs *config.DefaultTTLs, bidCategory map[string]string) []error {
-	if !bids && !vast {
+func (a *auction) doCache(ctx context.Context, cache prebid_cache_client.Client, targData *targetData, bidRequest *openrtb.BidRequest, ttlBuffer int64, defaultTTLs *config.DefaultTTLs, bidCategory map[string]string) []error {
+	var bids, vast, includeBidderKeys, includeWinners bool = targData.includeCacheBids, targData.includeCacheVast, targData.includeBidderKeys, targData.includeWinners
```

Comment:
This can simply be written as `bids, vast, includeBidderKeys, includeWinners := targData.includeCacheBids, targData.includeCacheVast, targData.includeBidderKeys, targData.includeWinners`. In Go, inside functions you can use the `:=` short assignment statement that will automatically infer the type of the values on the right and assign them to the variables on the left.

Reply:
I know, I just thought it's more readable this way. What do you think?

Reply:
I suppose Gus' way makes it explicit that they are booleans, but that should be easy to infer from the context below.

Reply:
@guscarreon If you think it is more readable this way then sure. I don't mind it 👍